pkgs/tools/nix/nixpkgs-manual-mmdoc-watch/default.nix

```diff
@@ -0,0 +1,19 @@
+{ writeScriptBin, python3, entr } :
+let
+  np-build = writeScriptBin "np-build" ''
+    nix build .#nixpkgs-manual-mmdoc -L
```

Comment:
This impurely depends on experimental-fearure nix-command which should be enabled with the command

Reply:
```suggestion
     nix-build . -A nixpkgs-manual-mmdoc
```

Reply:
Also, due to lazy trees not yet existing, this is plain slower than `nix build -f  . nixpkgs-manual-mmdoc` (if we *are* using experimental syntax) or indeed `nix-build -A nixpkgs-manual-mmdoc`.